source/io/module_javaScriptInvoke.js

```diff
@@ -79,28 +81,26 @@
 
         // Private Instance Variables (per vireo instance)
         var internalFunctionsMap = new Map();
-        var JavaScriptInvoke_GetParameterType = Module.cwrap('JavaScriptInvoke_GetParameterType', 'number', ['number', 'number']);
-        var JavaScriptInvoke_GetParameterPointer = Module.cwrap('JavaScriptInvoke_GetParameterPointer', 'number', ['number', 'number']);
-        var JavaScriptInvoke_GetArrayElementType = Module.cwrap('JavaScriptInvoke_GetArrayElementType', 'number', ['number']);
+        var JavaScriptInvoke_GetParameterTypeRef = Module.cwrap('JavaScriptInvoke_GetParameterTypeRef', 'number', ['number', 'number']);
+        var JavaScriptInvoke_GetParameterDataRef = Module.cwrap('JavaScriptInvoke_GetParameterDataRef', 'number', ['number', 'number']);
```

Comment:
Same as above, at the callsite use Module._JavaScriptInvoke_GetParameterDataRef instead

Reply:
Will do.